app/workers/podcasts/get_episodes_worker.rb

```diff
@@ -0,0 +1,22 @@
+module Podcasts
+  class GetEpisodesWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(podcast_data = {})
```

Comment:
the only drawback of this is that by removing the named parameters one has to read the method (or find out where it's used, which kind of helps as neither `limit`, nor `force_update` seem to be employed in the actual code, except in the specs) to understand what parameters should be given, maybe we can add a comment before the method explaining what is expected?

Reply:
That's fair, I'm down with that! Updated.